packages/@expo/cli/src/start/server/metro/__tests__/withMetroMultiPlatform.test.ts

```diff
@@ -1085,15 +1085,15 @@ describe(withExtendedResolver, () => {
       // 2: Resolves the origin root module path for `expo`
       expect(getResolveFunc()).toHaveBeenNthCalledWith(
         2,
-        expect.objectContaining({ originModulePath: 'expo/package.json' }),
+        expect.objectContaining({ originModulePath: '//package.json' }),
```

Comment:
These double `//` don't seem right, or is that expected?

Reply:
This is "expected" in that the origin module is set to a root path here: https://github.com/expo/expo/blob/7f93b0be2f9c890a48d93ec8f8a7e34956b75a5e/packages/%40expo/cli/src/start/server/metro/__tests__/withMetroMultiPlatform.test.ts#L69

So the resulting root base path is `/`, which isn't usually what would happen in a real app, but just looks weird here due to the mocks

Reply:
This might increase clarity but it's a wide-ranging change in the tests: https://github.com/expo/expo/pull/36181/commits/f8af0d20e7a22640295cb29b4d4634eed3ea650e